src/gui/accountmanager.cpp

```diff
@@ -531,4 +544,19 @@ void AccountManager::addAccountState(AccountState *accountState)
     _accounts << ptr;
     emit accountAdded(accountState);
 }
+
+bool AccountManager::forceLegacyImport() const
```

Comment:
@claucambra I do not see where this is used. Shouldn't you skip the popup if this is set?

Reply:
Oops, didn't push the commit -- the dialog is now skipped if this is true